test/testdata/schema/ghc98/default-config.golden.json

```diff
@@ -151,5 +151,6 @@
         "stan": {
             "globalOn": false
         }
-    }
+    },
+    "sessionLoading": "Single Component"
```

Comment:
This looks different from most of our other option values:
- None have spaces
- Most start lower-case
- The uppercase ones are clearly aeson-derived so look more like `SingleComponent`

It's all a bit inconsistent but I think this makes it _more_ inconsistent. So I'd go with `SingleComponent` or even better `singleComponent`.

Reply:
Ok, will change as suggested!